### ISSUE TITLE
Addition to config.yml for Oracle

### ISSUE THREAD

**Issue by AxxL** [mapbender/mapbender#693]
Provide an addition for Oracle and the config.yml where all Oracle-specific stuff can be placed. If you use Oracle you can simply import that yaml, for example like:

```yml
imports:
    - { resource: oracle.yml }
```

Content of the oracle.yml so far:

```yml

                services:
                  oracle.session.listener:
                    class: Doctrine\DBAL\Event\Listeners\OracleSessionInit
                    tags:
                      - { name: doctrine.event_listener, event: postConnect }
```

Adjust filename as it fits best to conventions.

**Comment by AxxL** [comment]
@werrolf @eSlider For your information. Any comments?

**Comment by eSlider** [comment]
@AxxL, @LazerTiberius strange but good solution!

**Comment by werrolf** [comment]
Resolved via pull #60
